paddle/fluid/operators/where_op.cu

```diff
@@ -0,0 +1,80 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
2016->2019
The following files should also be modified.

Reply:
done, thanks!